sqrl-examples/sensors/patient-sensor.sqrl

```diff
@@ -0,0 +1,40 @@
+IMPORT time.*;
+IMPORT patientpackage.Sensors TIMESTAMP epochMilliToTimestamp(placed) AS placedTimestamp;
+IMPORT patientpackage.SensorReading TIMESTAMP epochMilliToTimestamp(time) AS timestamp;
+--IMPORT patientapi.AddReading AS SensorReading TIMESTAMP _source_time;
+IMPORT patientpackage.ObservationGroup;
+
+/* Aggregate sensor readings to minute */
+MinReading := SELECT sensorid, endOfMinute(timestamp) as timeMin,
```

Comment:
Min reading sounds like minimum reading. Especially since we have a 'MaxSensorTemp' in the other sensor example.

Reply:
Also for timeMin, since we have a maxTemp a few lines down

Reply:
fixed